tests/unit/stateful-test.js

```diff
@@ -1,12 +1,244 @@
 import Ember from 'ember';
 import StatefulMixin from 'ember-stateful/mixins/stateful';
+import ERRORS from 'ember-stateful/errors';
 import { module, test } from 'qunit';
 
 module('Unit | Mixin | stateful');
 
 // Replace this with your real tests.
 test('it works', function(assert) {
```

Comment:
this is a really bad name for a test. Says absolutely nothing. What exactly are we testing here?

Reply:
that's the default test, I left it as it was generated. I'll rename it so it's more obvious what it does.